src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,38 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * <h1> Event </h1>
+ * Encapsulates a task that has a description and a date and time that it occurs on.
+ *
+ * @author Clifford
+ */
+public class Event extends Task {
+    protected final static String taskSymbol = "[E]";
```

Comment:
Perhaps taskSymbol can be changed to `TASK_SYMBOL` as it is a constant.

Reply:
Thank you for taking time off your busy schedule to review my code. 
I have taken note of your comments and will make the necessary changes 😄